applications/IgaApplication/iga_application.cpp

```diff
@@ -49,6 +52,7 @@ void KratosIgaApplication::Register() {
 
     KRATOS_REGISTER_VARIABLE(CROSS_AREA)
     KRATOS_REGISTER_VARIABLE(PRESTRESS_CAUCHY)
+    KRATOS_REGISTER_VARIABLE(PRESTRESS)
```

Comment:
you can use KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS instead. This defines you immediately that size is 3. That is much more efficient.

Further, you need to differentiate if your prestress is cauchy or pk or pk2.